pallets/parachain-staking/src/lib.rs

```diff
@@ -1865,13 +1909,18 @@ pub mod pallet {
 			}
 			collators.sort();
 
+			<Total<T>>::mutate(|old| {
```

Comment:
I would rename `old` to something else, as we are using the parameter to update the total stake, and it could be misleading.